### PR TITLE
Adds exclusion for words that contain green

### PR DESCRIPTION
**Pull request by wrkode**
This PR adds logic for excluding words and last names that contain Green.

### CODE REVIEW

**Comment by leonardpahlke** [comment]
cc @Al-HusseinHameedJasim

File: greenscraper/exclusions.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Greenleaf
+Greenlee
+Greenspan
+Greenstein
+Greenwald
+Greenwell
+Greenblatt
+Greenberg
+Greenfield
+Greenbaum
```
Comment (leonardpahlke) [inline greenscraper/exclusions.txt:41]: I am wondering if we can just match on `" Green "` so anything that has green as part of a word does not get matched

**Comment by leonardpahlke** [comment]
PTAL @wrkode

File: greenscraper/exclusions.txt (hunk 1 above)
Comment (wrkode) [reply greenscraper/exclusions.txt:41]: The issue is that there are several words that contain "green" and are relevant to the search/topic

Comment (wrkode) [reply greenscraper/exclusions.txt:41]: I could have it as a "TODO" to explore optimization in the future?

Comment (leonardpahlke) [reply greenscraper/exclusions.txt:41]: sounds good

**Review by leonardpahlke — APPROVED** [review]
LGTM, thanks!

**Pull request merged**